music/src/core/player.ts

```diff
@@ -78,13 +79,27 @@ export abstract class BasePlayer {
    *   @param callbackObject An optional BasePlayerCallback, specifies an
    *     object that contains run() and stop() methods to invode during
    *     playback.
+   *   @param bpm An optional number specifying the starting BPM for playback.
    */
-  constructor(playClick = false, callbackObject?: BasePlayerCallback) {
+  constructor(playClick = false, callbackObject?: BasePlayerCallback,
+              bpm = 60) {
     this.playClick = playClick;
     this.callbackObject = callbackObject;
     // Set a bpm of 60 to make dealing with timing easier. We will use seconds
     // instead of transport time since it can't go beyond 16th notes.
-    Tone.Transport.bpm.value = 60;
+    this.desiredBPM = bpm;
+  }
+
+  /**
+   * Changes the bpm of the playback.
+   *
+   * @param bpm A number, the new bpm to use.
+   */
+  setBPM(bpm: number) {
```

Comment:
We use `qpm` elsewhere, and we should be consistent. Maybe calling this `setTempo(qpm: number)` would be even more clear?

Reply:
done.